### ISSUE TITLE
Example server should not use port 9000 by default

### ISSUE THREAD

**Issue by bwl21** [voidlabs/mosaico#69]
The example server in main.js uses port 9000. This port ist also used by default for the PHP debugger xdebug. When debugging a system with those default settings this causes problems. By whatever magic main.js uses next free port (e.g. 9001)  if 9000 is already occupied (e.g. if a PHP debug session is active). But this leads to the fact that we have another localstorage and the newslettes of localhost:9000 are no longer accessible.

As I somtimes run example server at the same time as a php based server on my localhost, I get conflicts.

Long story short: life would be easier if example server would use a port which is not listed on https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers#Well-known_ports.
